extensions/gc/tasks/GarbageCollectorTask.js

```diff
@@ -23,16 +23,16 @@ class GarbageCollectorTask extends BackbeatTask {
     // account id information is missing.
     // TODO BB-367: replace once replication services uses assume role
     _getAccountId(entry, log, cb) {
-        const { accountId, ownerId } = entry.getAttribute('target');
+        const { accountId, owner } = entry.getAttribute('target');
```

Comment:
also needs to be changed in https://github.com/scality/backbeat/blob/c790d6de0e52b1ba701dd1fe7103362fb97ed093/extensions/replication/tasks/UpdateReplicationStatus.js#L279